spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -190,8 +189,8 @@ def get_verterbral_level_from_slice(slices, df_metrics):
                              f"Check vertebral labeling file.")
     level_slice_dict = {}
     # TODO adjust for multiple slices for one compresssion (that can have multiple levels too)
-    #slices_combined = []
-    #for slice in slices:
+    # slices_combined = []
+    # for slice in slices:
```

Comment:
Looks like @mguaypaq's PR https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4161 has caught a [linting issue](https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/5671517730/job/15368809512) that slipped through!

(Thanks again for fixing the linting workflow! :hearts:)